src/Core/Grid/Definition/Factory/ProductGridDefinitionFactory.php

```diff
@@ -337,4 +399,38 @@ protected function getGridActions()
                     ->setIcon('storage')
             );
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getBulkActions()
```

Comment:
I think all bulk actions should be handled in a separate PR ith the modal and Ajax calls